department-of-veterans-affairs/caseflow
# Colocated | Send back to assigning attorney

Connects #6086 

### Description
This adds a `SendToAssigningAttorneyModal` which is displayed when Send to Attorney is selected from the `ColocatedActionsDropdown` in `CaseDetailsView`. This also replaces `queue/constants.USER_ROLES` with `USER_ROLE_TYPES.json`

### Acceptance Criteria 

### Testing Plan
1. Go to `/queue` as a colocated user (`BVALSPORER`), go to case details screen
2. Choose Send back to Attorney from dropdown, confirm modal appears
3. Confirm assignment happens successfully / user is returned to `/queue` on success

<details><summary>gif</summary>
<img src="https://user-images.githubusercontent.com/8533004/44153515-6b43beb4-a076-11e8-9ff8-6a64a10fff34.gif">
</details>
